src/ui/components/LedgerViewer.js

```diff
@@ -99,21 +105,29 @@ export const LedgerViewer = ({
             <TableRow>
               <TableCell>Event</TableCell>
               <TableCell>Details</TableCell>
-              <TableCell align="right">Date</TableCell>
+              <TableCell align="right">
+                <TableSortLabel
+                  active={ts.sortName === DATE_SORT.name}
+                  direction={
+                    ts.sortName === DATE_SORT.name ? ts.sortOrder : DEFAULT_SORT
```

Comment:
Why do we only use the sortOrder from TableState when its Date and not any other filter?

Reply:
Each TableSortLabel corresponds to 1 column. The sortOrder applies only to the current sortName. Although there is only 1 sortable column in this PR, there could be others later. If this column is not the actively sorted column, then the button should show the default direction (the direction that the column will sort to upon becoming the actively sorted column).

The easiest way to understand is probably trying out the UX for yourself.